orchestrator/cosmos_gravity/src/send.rs

```diff
@@ -126,16 +126,20 @@ pub async fn send_messages(
     gas_adjustment: f64,
 ) -> Result<TxResponse, GravityError> {
     let cosmos_address = cosmos_key.to_address(&contact.get_prefix()).unwrap();
+    let min_gas_limit = 500000 * messages.len() as u64;
 
+    // compute the fee as fee=ceil(gas_limit * gas_price)
+    let fee_amount: f64 = min_gas_limit as f64 * gas_price.0;
+    let fee_amount: u64 = fee_amount.abs().ceil() as u64;
     let fee_amount = Coin {
         denom: gas_price.1.clone(),
-        amount: 0u32.into(),
+        amount: fee_amount.into(),
```

Comment:
cosmos-sdk 0.46 reject 0 gas tx.

Reply:
reverted the change, will be fixed at cosmos-sdk side: https://github.com/cosmos/cosmos-sdk/pull/12590